buildroot/share/PlatformIO/scripts/stm32_serialbuffer.py

```diff
@@ -0,0 +1,14 @@
+#
+# stm32_serialbuffer.py
+#
+Import("env")
+
+# Set the buffer size according to the configuration file or 64 if too small or absent.
+mf = env["MARLIN_FEATURES"]
```

Comment:
I think you can add one more info here too:

"We need take Marlin rx/tx buffer size and pass it using global build flags because it's the only way to change internal stm32duino rx/tx buffer sizes"... something like that.